tutorials/advanced_features/Virtual Devices.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>Remember that devices are generally immutable, so directly linking your variable to a built-in device is not a good idea. I would suggest you use <code>replace</code> again to show you can start from <code>MockDevice</code> and make a few tweaks to have your own mock device.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>